src/style.css

```diff
@@ -1,3 +1,78 @@
+html {
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
It's a good practice to provide fallback fonts in case the main one doesn't load. For example, you could modify the `font-family` property to `font-family: 'Roboto', Arial, sans-serif;`. This way, if the 'Roboto' font fails to load, the browser will try to load 'Arial', and if that fails, it will default to any sans-serif font available on the user's system.